scripts/optimize.py

```diff
@@ -53,6 +51,17 @@ def get_emission_limit():
         return limit
 
 
+def get_additional_scalars():
+    """Returns additional scalars as pd.DataFrame or None"""
+    filename_add_scalars = os.path.join(preprocessed, "additional_scalars.csv")
+    if os.path.exists(filename_add_scalars):
+        scalars = dp.load_b3_scalars(filename_add_scalars)
+        emission_limit = get_emission_limit(scalars)
+    else:
+        emission_limit = None
+    return emission_limit
```

Comment:
I suggest to decouple `get_additional_scalars` and `get_emission_limit`. 

In the main part of the script, let `get_additional_scalars` load the scalars and nothing more. If it returns something, call  `get_emission_limit`, passing the additional scalars. If not, do not process the emission limit.

Reply:
done in [e8ca7ab](https://github.com/rl-institut/oemof-B3/pull/153/commits/e8ca7ab67a4ddf9a6b339d6b422817f0ac1b1689)

this leads to a warning in PyCharm that `emission_limit` can be undefined. Which I ensure not to be, as I first ask whether `get_additional_scalars` returned something and if it did `emission_limit` exists. But I am not sure whether this is good coding practice.